dev-guide/requirements.md

```diff
@@ -0,0 +1,31 @@
+# Build requirements
+
+Learn about the following requirements before building TiDB.
+
+## Supported platforms
+
+The following table lists TiDB support for common architectures and operating systems.
+
+|Architecture|Operating System|Status|
+|:------------|:----------------|:------|
+|AMD64|Linux Ubuntu (14.04+)|Stable|
+|AMD64|Linux CentOS (7+)|Stable|
+|AMD64|macOS|Experimental|
+
+## Prerequisites
+
++ Go [1.12+](https://golang.org/doc/install)
++ Rust [nightly version](https://www.rust-lang.org/downloads.html)
```

Comment:
I think what's required is rustup instead of `Rust`, and rustup will download and setup the proper version of rust compiler for tikv.

Reply:
Updated.